tests/spread/dependencies/strict-dependencies-errors/extra-packages/charmcraft.yaml

```diff
@@ -0,0 +1,13 @@
+name: charm
+type: charm
+title: Strict dependencies test charm
+summary: a test charm for checking that strict dependencies are used.
+description: a test charm for checking that strict dependencies are used.
```

Comment:
suggestion: use the description to record why exactly this charm should fail to pack

Reply:
Good idea! I blindly copy/pasted these from the success case and didn't think about updating that metadata.

I've also added another variant of this test that checks for extra packages in `charm-python-binary-packages`, since that should also be a subset of the packages mentioned in requirements.txt.